trajopt/src/collision_terms.cpp

```diff
@@ -1473,9 +1530,8 @@ void CastCollisionEvaluator::Plot(const std::shared_ptr<tesseract_visualization:
     dist_results_copy[i] = res;
 
     // Contains the contact distance threshold and coefficient for the given link pair
-    const std::array<double, 2>& data =
-        getSafetyMarginData()->getPairSafetyMarginData(res.link_names[0], res.link_names[1]);
-    safety_distance[i] = data[0];
+    safety_distance[i] = collision_config_->contact_manager_config.margin_data.getPairCollisionMargin(
+        res.link_names[0], res.link_names[1]);
```

Comment:
Same comment about renaming to `collision_margins`

Reply:
Updated